dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventChangeLogService.java

```diff
@@ -45,6 +48,21 @@ Page<EventChangeLog> getEventChangeLog(
       UID event, EventChangeLogOperationParams operationParams, PageParams pageParams)
       throws NotFoundException, ForbiddenException;
 
+  /**
+   * @deprecated use {@link EventChangeLogService#getEventChangeLog} instead
+   */
+  @Deprecated(since = "2.41")
+  List<org.hisp.dhis.trackedentitydatavalue.TrackedEntityDataValueChangeLog>
+      getTrackedEntityDataValueChangeLogs(TrackedEntityDataValueChangeLogQueryParams params);
+
+  void addTrackedEntityDataValueChangeLog(
```

Comment:
Should we not also deprecate these 3 methods?

Reply:
we don't have a replacement for the other ones yet, and I'm not sure we actually need to replace them at all.
this one was deprecated because of the new change log feature, which returns the data in a different format, but the other methods (to save and delete change logs) are still useful the way they are.

Reply:
that one we can deprecate, I'll do it in the following PR